<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTI_ALPS</class>
 <widget class="qMRMLWidget" name="DTI_ALPS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>311</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Inputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="inputDTILabel">
        <property name="text">
         <string>Input DTI Volume</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputDTISelector" native="true">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputVolume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="inputProjLabel">
        <property name="text">
         <string>Input Projection Label</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="inputProjLabelSelector" native="true">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
          <string>vtkMRMLLabelMapVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputVolume</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="inputAssocLabel">
        <property name="text">
         <string>Input Association Label</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="inputAssocLabelSelector" native="true">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
          <string>vtkMRMLLabelMapVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputVolume</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Advanced</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="checkMNISpace">
        <property name="text">
         <string>Input is in MNI space (2 mm resolution)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="MNISpaceCheckBox">
        <property name="toolTip">
         <string>If checked, values above threshold are set to 0. If unchecked, values below are set to 0.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="invertThreshold" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DTI-ALPS index</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dti_alps_output"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Calculate DTI-ALPS Index</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>DTI_ALPS</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputDTISelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DTI_ALPS</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputAssocLabelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DTI_ALPS</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputProjLabelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
